packages/mco/components/disaster-recovery/applications/failover-and-relocate-modal/dr-policy-selector.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import { useDeepCompareMemoize } from '@odf/shared/hooks/deep-compare-memoize';
+import { getName } from '@odf/shared/selectors';
+import { k8sGet } from '@openshift-console/dynamic-plugin-sdk';
+import { TFunction } from 'i18next';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import { DRPolicyModel } from '../../../../models';
+import { DRPolicyKind } from '../../../../types';
+import { getErrorMessage } from '../../../../utils';
+import {
+  ACMToDRState,
+  FailoverAndRelocateState,
+  FailoverAndRelocateAction,
+  FailoverAndRelocateType,
+  DRPolicyType,
+} from './reducer';
+
+const getDRPolicyList = async (acmToDRState: ACMToDRState[]) => [
```

Comment:
```suggestion
const getDRPolicyList = (acmToDRState: ACMToDRState[]) => [
````
`getDRPolicyList` is not handling any Promise...